app/views/users/edit.html.erb

```diff
@@ -63,7 +63,7 @@
 
   <div class="crayons-layout__content">
     <% if current_user.email.blank? %>
-      <div class="crayons-notice crayons-notice--warning mb-6">
+      <div class="crayons-notice crayons-notice--warning mb-6" role="alert">
```

Comment:
```suggestion
      <div class="crayons-notice crayons-notice--warning mb-6" aria-live="polite">
```

refactor (blocking) - it's important but it's not an error so let's make this polite